components/result-count-title/package.json

```diff
@@ -23,10 +23,11 @@
   },
   "main": "lib/index.js",
   "module": "es/index.js",
-  "author": "Tarandeep Singh Chauhan",
+  "author": "Steve Sims",
```

Comment:
this one might be me - can't remember exactly - plain "The govuk-frederic authors" may be best